packages/truffle-compile/profiler/getImports.js

```diff
@@ -0,0 +1,21 @@
+const { isExplicitlyRelative } = require("./isExplicitlyRelative");
+const Parser = require("../parser");
+const path = require("path");
+
+const getImports = (file, { body, source }, solc) => {
+  // No imports in vyper!
+  if (path.extname(file) === ".vy") return [];
```

Comment:
heh it's kinda silly that this is here... but vyper _is_ using the profiler, I know that.